<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
 background :  ; 
 border-radius : 60px ; 
}
QLabel
{
 color : #48494B  ; 
}
QLineEdit
{
 background:transparent ; 
border : none ;
color:#48494B ;
font-size : 20px ; 
border-bottom:0.5px solid #717072 ; 
}
QTabWidget::pane {
  top:-40px; 
} 
QTabBar::tab {
background-color: rgb(217, 221, 220);
border-radius : 15px;
border: 2px solid lightgray; 
border-color: rgb(255, 255, 255);
padding: 3px;
} 

QTabBar::tab:selected { 
background-color: rgb(217, 221, 220);
border: 3px solid lightgray; 
border-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 140, 0);
border-right-color: rgb(255, 140, 0);
margin-bottom: 0px; 

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1181</width>
      <height>681</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(217, 221, 220);
border-radius : 35px ; </string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_CreateEmp">
     <attribute name="title">
      <string>Create Employe</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputname">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>190</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="service">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>460</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Service</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputlastname">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>190</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>450</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputcmdp">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>400</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Confirm Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>450</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdp">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Cnx">
     <attribute name="title">
      <string>Se Connecter</string>
     </attribute>
     <widget class="QLineEdit" name="emailcnx">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdp_cnx">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>140</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginbutton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>440</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_EmpManagment">
     <attribute name="title">
      <string>Emloye Managment</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>341</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>List Employes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdfinput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>190</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Fournisseurs/Icons/search.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchinput">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>140</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sortinput">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Statistics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updatebutton">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>480</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletebutton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>480</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QTableView" name="listeEmploye">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>1021</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Menu">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>20</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Statistics">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QWidget" name="statwidget" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>621</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="testmedicament">
     <attribute name="title">
      <string>service medical</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>471</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Service Medical</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_chat">
     <attribute name="title">
      <string>Discussion</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>160</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="listeUsers"/>
     </widget>
     <widget class="QPushButton" name="sendMessage">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>610</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textmsg">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>501</y>
        <width>501</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="nomre">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>80</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>501</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="listeMsg"/>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>En Ligne</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="formulareemploye">
     <attribute name="title">
      <string> Formulaire Employé</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="fe_name">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fe_lastname">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>110</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fe_email">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fe_password">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>250</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fe_confirm_password">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Confirm Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fe_service">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>410</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Service</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>470</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>540</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
